internal/elasticsearch/index/index_test.go

```diff
@@ -83,6 +83,7 @@ func TestAccResourceIndexSettings(t *testing.T) {
 					resource.TestCheckResourceAttr("elasticstack_elasticsearch_index.test_settings", "routing_rebalance_enable", "primaries"),
 					resource.TestCheckResourceAttr("elasticstack_elasticsearch_index.test_settings", "gc_deletes", "30s"),
 					resource.TestCheckResourceAttr("elasticstack_elasticsearch_index.test_settings", "analysis_analyzer", `{"text_en":{"filter":["lowercase","minimal_english_stemmer"],"tokenizer":"standard","type":"custom"}}`),
```

Comment:
This expectation also needs to be updated:

```
index_test.go:50: Step 1/1 error: Check failed: Check 28/32 error: elasticstack_elasticsearch_index.test_settings: Attribute 'analysis_analyzer' expected "{\"text_en\":{\"filter\":[\"lowercase\",\"minimal_english_stemmer\"],\"tokenizer\":\"standard\",\"type\":\"custom\"}}", got "{\"text_en\":{\"char_filter\":\"zero_width_spaces\",\"filter\":[\"lowercase\",\"minimal_english_stemmer\"],\"tokenizer\":\"standard\",\"type\":\"custom\"}}"
```

Reply:
Sorry, fixed!
https://github.com/elastic/terraform-provider-elasticstack/pull/158/commits/b67b1a9808a7ac0409a1adcc04683f183cf2d2e8